x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/settings/agent_configurations.spec.ts

```diff
@@ -101,4 +101,29 @@ describe('Agent configuration', () => {
     cy.wait('@serviceEnvironmentApi');
     cy.contains('production');
   });
+  it('displays All label when selecting all option', () => {
+    cy.intercept(
+      'GET',
+      '/api/apm/settings/agent-configuration/environments?serviceName=ENVIRONMENT_ALL'
```

Comment:
Could `ENVIRONMENT_ALL` be renamed to something else as it's used as a service name here?

Reply:
my bad with the previous commit  I introduced a bug. 
it should [omit](https://github.com/elastic/kibana/pull/133616/files/bea11c6acf493a89c6f18419c1e6a886893b8278#diff-b23eb2353f30e68304959fb4d0def2c1f80c45d552c4d6f7fb4d8f5668ff185aL24) the `serviceName` when you select All in order to fetch all environments. 

updated [here](https://github.com/elastic/kibana/pull/133616/commits/cf04974141e1241ec1c15a7ad627005007b1b6f6)